test/mocha/35-operations-api.js

```diff
@@ -76,19 +72,17 @@ describe('Operations API', () => {
           await Ed25519VerificationKey2020.from(mockData.keys.authorized);
         const operation = await helpers.signDocument({
           doc: testOperation,
-          verificationMethod:
-            'did:v1:uuid:53ebca61-5687-4558-b90a-03167e4c2838#keys-144',
           key
         });
         await ledgerNode.operations.add({operation});
       });
       it('should fail add operation with an incorrect context', async () => {
         const testOperation = {
-          '@context': 'https://w3id.org/test/v1',
+          '@context': constants.TEST_CONTEXT_V1_URL,
           type: 'CreateWebLedgerRecord',
           creator: 'https://example.com/someCreatorId',
           record: {
-            '@context': 'https://schema.org/',
+            '@context': constants.TEST_CONTEXT_V1_URL,
             id: 'urn:uuid:' + uuid(),
             value: uuid()
```

Comment:
If the test context is going to be used here then `value` needs to be defined in that context.

Reply:
Changed value to name: https://github.com/digitalbazaar/bedrock-ledger-node/commit/f59451ec5828d97289c38c2eec252b3f842ed8bc